wolfcrypt/src/evp.c

```diff
@@ -656,6 +673,63 @@ static int wolfSSL_EVP_CipherUpdate_GCM(WOLFSSL_EVP_CIPHER_CTX *ctx,
 }
 #endif /* HAVE_AESGCM */
 
+#if defined(HAVE_AESCCM)
+static int wolfSSL_EVP_CipherUpdate_CCM_AAD(WOLFSSL_EVP_CIPHER_CTX *ctx,
+        const unsigned char *in, int inl) {
+    if (in && inl > 0) {
+        byte* tmp = (byte*)XREALLOC(ctx->gcmccmAuthIn,
```

Comment:
For the rename to `gcmccm` perhaps consider since these are for authenticated ciphers something like:
* `gcmccmAuthIn` -> `authIn`
* `gcmccmAuthInSz` -> `authInSz`
* `gcmccmBuffer` -> `authBuffer`
* `gcmccmBufferLen` -> `authBufferLen`